Tiltfile

```diff
@@ -390,7 +390,7 @@ def get_addons(flavor_name):
     else:
         calico_values = "./templates/addons/calico/values.yaml"
 
-    addon_cmd = "; " + helm_cmd + " --kubeconfig ./${CLUSTER_NAME}.kubeconfig install --repo https://docs.tigera.io/calico/charts calico tigera-operator -f " + calico_values + " --namespace tigera-operator --create-namespace"
+    addon_cmd = "; CALICO_VERSION=$(make get-calico-version) " + helm_cmd + " --kubeconfig ./${CLUSTER_NAME}.kubeconfig install --repo https://docs.tigera.io/calico/charts --version ${CALICO_VERSION} calico tigera-operator -f " + calico_values + " --namespace tigera-operator --create-namespace"
```

Comment:
Is this a new pattern for storing env vars in one place? (Makefile)

Reply:
Is this not what you meant by https://github.com/kubernetes-sigs/cluster-api-provider-azure/pull/3369/files#r1199457600 ?